### ISSUE TITLE
Status bar elements not accessible through command pallet

### ISSUE THREAD

**Issue by Entomy** [microsoft/vscode#93494]
- VSCode Version: 1.43.0
- OS Version: Windows_NT x64 10.0.18363

Does this issue occur when all extensions are disabled?: **Yes**

Despite being reported and discussed two years ago with #51977 there are in fact status bar elements not part of extensions that are not accessible through the command pallet, and are therefore entirely inaccessible. Specifically, the feedback control.

This is particularly concerning because this is part of the mechanisms I was told are used to actually get information from affected individuals. So if one of the mechanisms to get feedback isn't accessible by the people who would need to give that critical feedback, then... Oh boy.

**Comment by isidorn** [comment]
Agree with you.
The tweet feedback is not accessible via the command palette, we should also add it there so user can access it via command palette.
@bpasero do you mind if I add the tweet feedback command to the command palette. Not sure who own the tweet feedback widget?

@Entomy thanks for filling this issue. If there are other parts of the status bar which are not accessible please let me know and we will fix them. For now only plan to fix the tweet feedback command.

**Comment by Entomy** [comment]
@isidorn That's the only one on this build. We checked all non-extension parts of the status bar.

I'd love to let you know about others but I can't use your product anymore, as described in a separate issue which has been ignored. I had my girlfriend help me with checking this because of a discussion elsewhere.

**Comment by bpasero** [comment]
@isidorn I was actually wondering why we are not putting it both to the help menu as well as the command palette, I think it could be in both places.

**Comment by isidorn** [comment]
I have added the `Help: Tweet Feedback` to the command palette. Thus you can now access it via `F1`.
Also the command id is `help.tweetFeedback` so you can assign a custom keybinding to it.

@Entomy I am sorry to hear that you can not use the produce anymore. If you can create more issues with steps like this one that would be great because then we can look into them and fix them. If you are having a hard time creating issues my collegue @sana-ajani and me can do a online call with you so you try to explain your issues with the product and we try to create as many GitHub issues as possible which we could tackle in the future. Thanks a lot!
